### ISSUE TITLE
Mode2Alpha: LUA Callstack on Load with Dominator 108 SL Maxi

### ISSUE THREAD

**Issue by Merrlin** [Courseplay/courseplay#4762]
Hi @pvaiko
 
Bad News. I've downloaded the latest (02.01.2020 v.355) CombineUnloadAIDriver branch. When I load the the Savegame the Game hangs totallywith anLUA Callstack when I hit enter to start the game Itself. The following LUA Callstack occurs:

2020-01-02 19:59 LUA call stack:
2020-01-02 19:59   =C:/Users/Daniel/Documents/My Games/FarmingSimulator2019/mods_kaif/FS19_Courseplay/CombineUnloadManager.lua (331) : getParent
2020-01-02 19:59   =C:/Users/Daniel/Documents/My Games/FarmingSimulator2019/mods_kaif/FS19_Courseplay/CombineUnloadManager.lua (299) : getPipesBaseNode
2020-01-02 19:59   =C:/Users/Daniel/Documents/My Games/FarmingSimulator2019/mods_kaif/FS19_Courseplay/CombineUnloadManager.lua (352) : getPipeOffset
2020-01-02 19:59   =C:/Users/Daniel/Documents/My Games/FarmingSimulator2019/mods_kaif/FS19_Courseplay/CombineUnloadManager.lua (191) : getOnFieldSituation
2020-01-02 19:59   =C:/Users/Daniel/Documents/My Games/FarmingSimulator2019/mods_kaif/FS19_Courseplay/CombineUnloadManager.lua (181) : updateCombinesAttributes
2020-01-02 19:59   =C:/Users/Daniel/Documents/My Games/FarmingSimulator2019/mods_kaif/FS19_Courseplay/CpManager.lua (269) : onUpdate
2020-01-02 19:59   dataS/scripts/BaseMission.lua (1599) : update
2020-01-02 19:59   dataS/scripts/FSBaseMission.lua (1977) : update
2020-01-02 19:59   dataS/scripts/missions/mission00.lua (469) : update
2020-01-02 19:59   dataS/scripts/main.lua (1896) : update
2020-01-02 19:59 Warning (script): 'getName': Argument 1 has wrong type. Expected: Int. Actual: Nil 
2020-01-02 19:59   ..\..\..\src\base\scripting\lua\LuaScriptSystem.cpp (882): expectedType == Value::VoidType || expectedType == Value::ClassType

**Comment by Tensuko** [comment]
Well, it is an Alpha for a reason...
There is still several things to do...
I can use it on my Testsave (Vanilla) just with some turn Issue of the Mode2 Driver.

**Comment by pvaiko** [comment]
As we all know, we need to see a full logfile...

**Comment by Merrlin** [comment]
Yes of course I know its an Alpha... Logile is shortened because of the LUA Callstack repeats literally a thousand times as long you don't kill the App via TaskManager (must be on Second screen).
On the Felsbrunn with no Mods everything is fine (or at least it loads).
[log.txt](https://github.com/Courseplay/courseplay/files/4017589/log.txt)

**Comment by pvaiko** [comment]
If you have combine mods I'd start disabling them one by one as it seems to be a problem with a combine pipe.

**Comment by Merrlin** [comment]
No combine mods in the mod folder. The only ones I've activated are the ones from the claas dlc including the dominator. The dominator is the one I've brought on this Savegame.
I've suspected the dominator for now but tonight there's no more time for testing. I will have a closer look tomorrow. The only thing that looks strange to me is that the dominator works on a testsave (at least it starts on a new game)
